json_serializable/test/kitchen_sink/kitchen_sink.dart

```diff
@@ -12,6 +12,7 @@ import 'strict_keys_object.dart';
 part 'kitchen_sink.g.dart';
 
 List<T> _defaultList<T>() => null;
```

Comment:
Can we get a comment here that these get overwritten to not return null in some cases :D

Reply:
Alternatively could you use like a zone variable or something to make this a bit less hacky?

Reply:
I'll add a comment. Let's keep the runtime clean.